src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -84,9 +93,13 @@ public HostOutputMessage HostOutputMessage
                 StartTime = functionStartedMessage.StartTime.DateTime
             };
 
+            Stopwatch sw = Stopwatch.StartNew();
             try
             {
-                functionStartedMessageId = await ExecuteWithLoggingAsync(functionInstance, functionStartedMessage, fastItem, parameterLogCollector, functionTraceLevel, cancellationToken);
+                using (_logger?.BeginScope(CreateLogScope(functionInstance.Id, functionInstance.FunctionDescriptor.Method.Name)))
```

Comment:
I wonder if you shouldn't add your own WebJobs specific extension methods, so you can simply do _logger?.BeginFunctionScope(...), since I see this pattern below as well.